### ISSUE TITLE
Seems the property showInTeamsSearchAndSuggestions is not working

### ISSUE THREAD

**Issue by TobiasAT** [microsoftgraph/microsoft-graph-docs-contrib#8964]
I tried to change the property showInTeamsSearchAndSuggestions for several private teams, but the value False does not change to True. 
No error with the request, just no change. 

Here is an example:
1. I check the current value of showInTeamsSearchAndSuggestions for a private Team
2. I change the value to True
3. I check the updated value of showInTeamsSearchAndSuggestions, but still False

![image](https://github.com/microsoftgraph/microsoft-graph-docs-contrib/assets/7470044/90e131ff-1484-44f8-acb2-92c054d39270)

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 736e99b0-0f0a-0644-8d4b-cfd2b246c0a1
* Version Independent ID: e6641dc7-a134-54de-85d2-7e7f66acb1fa
* Content: [teamDiscoverySettings resource type - Microsoft Graph beta](https://learn.microsoft.com/en-us/graph/api/resources/teamdiscoverysettings?view=graph-rest-beta)
* Content Source: [api-reference/beta/resources/teamdiscoverysettings.md](https://github.com/microsoftgraph/microsoft-graph-docs-contrib/blob/main/api-reference/beta/resources/teamdiscoverysettings.md)
* Product: **microsoft-teams**
* Technology: **microsoft-graph**
* GitHub Login: @nkramer
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by Prasad-MSFT** [comment]
Hi @TobiasAT - Could you please share the repro steps such as which API you used to update the property along with the request body, so that we can repro it from our side once?

**Comment by TobiasAT** [comment]
@Prasad-MSFT Sure, I used the API to update Teams: https://learn.microsoft.com/en-us/graph/api/team-update?view=graph-rest-beta&tabs=http

Here is the body (code formatting is because of the GitHub comment editor). 

`
$Body = @"
{
     "discoverySettings": { "showInTeamsSearchAndSuggestions": true  }
}
"@  
`

The body is also part of the example in the Update API. 

As addition, I also tried the change via the Teams PowerShell Module and [Set-Team](https://learn.microsoft.com/en-us/powershell/module/teams/set-team?view=teams-ps#-showinteamssearchandsuggestions). The result is the same like with the Update API. No changed with the property. As the parameter ShowInTeamsSearchAndSuggestions is marked as deprecated I used the Update API from Graph.  

I tested the same commands some minutes ago. Still the same situation.
